### PR TITLE
Rewrite README

### PR DESCRIPTION
**Pull request by snmgian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,269 +25,221 @@
...
-  config.on(:startup) do
-    Sidekiq.schedule = YAML.load_file(File.expand_path('../../scheduler.yml', __FILE__))
-    Sidekiq::Scheduler.reload_schedule!
-  end
-end
+See https://github.com/jmettraux/rufus-scheduler for more information.
+
+## Load the schedule from a different file
+
+You can place the schedule configuration in a separate file than `config/sidekiq.yml`
```
Comment (migl) [inline README.md]: "... place the schedule configuration in a separate file **from** `config/sidekiq.yml`"

File: README.md [hunk 2]
```diff
@@ -25,269 +25,221 @@
...
+clear_leaderboards_contributors:
+  cron: '0 30 6 * * 1'
+  class: ClearLeaderboards
+  queue: low
+  args: contributors
+  description: 'This job resets the weekly leaderboard for contributions'
 ```
 
-If you are running a non Rails project you should add code to load the workers classes before loading the schedule.
+Please note that `schedule` root key is not present in the separate file.
```
Comment (migl) [inline README.md]: "Please **notice** that **the** `schedule` root key is not present..."

File: README.md [hunk 3]
```diff
@@ -25,269 +25,221 @@
...
 
-``` ruby
-# config/initializers/sidekiq_scheduler.rb
-require 'sidekiq/scheduler'
+## Configuration options
 
-puts "Sidekiq.server? is #{Sidekiq.server?.inspect}"
-puts "defined?(Rails::Server) is #{defined?(Rails::Server).inspect}"
-puts "defined?(Unicorn) is #{defined?(Unicorn).inspect}"
+Configuration options are placed inside `sidekiq.yml` config file.
```
Comment (migl) [inline README.md]: I'd clarify that they are placed here by default so that it leads to the part where you explain how to use a different config file.

Comment (snmgian) [inline README.md]: These options can only be placed in sidekiq config file. They can not be placed in a separate file.

**Pull request merged**